### ISSUE TITLE
Fix WSGI links in documentation.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5882]
|<img alt="faulkner's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| faulkner reported|
|-|-|
|Trac ID|trac#5882|
|Type|defect|
|Created|2012-08-19 19:04:59Z|

WSGI moved to RTD.

Patch attached.

Attachments:

* [fix-wsgi-links.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/84e/84eda92fd1a59b443121e63fb57befe52f7b1822/f1ce6eeb5d821787c213ee05ff237a7d5407a2a8.patch) (1312 bytes) - added by faulkner on 2012-08-19 19:05:24Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__5882 5882
type__defect defect
reporter__faulkner faulkner
priority__low low
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1345403099000000 1345403099000000
changetime__1345570713000000 1345570713000000
version__None None
owner__thijs thijs

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_1"></a>|@thijstriemstra set owner to @thijstriemstra<br>@thijstriemstra set status to `assigned`|
|-|-|

Thanks for your patch. Make sure to add the `review` keyword when you propose a patch as described on ReviewProcess#Authors:Howtogetyourchangereviewed

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_2"></a>|@thijstriemstra commented|
|-|-|

wsgi.org redirects to the same new url in your patch, so I'll use that instead.

**Comment by twisted-trac** [comment]
|[<img alt="thijstriemstra's avatar" src="https://avatars.githubusercontent.com/u/305679?s=50" width="50" height="50">](https://github.com/thijstriemstra)<a name="note_3"></a>|@thijstriemstra set status to `closed`|
|-|-|

(In [35327]) Apply fix-wsgi-links.patch: Fix broken links for the WSGI documentation, available at readthedocs.org.

Author: faulkner
Reviewer: thijs
Fixes: [#5882](https://github.com/twisted/twisted/issues/5882)
